File: Unicatólica/Semestre_2024_2/Estrutura_de_Dados/Turma A/Codigos/aula_22102024/Q1.cpp
#include <iostream>
using namespace std;

int potencia(int base, int expoente){
    int resultado = 1;
    for(int i = 1; i<=expoente; i++){
        resultado = resultado * base;
    }
    return resultado;
}
int main(){
    int base, expoente;
    cout << "Digite o valor da base: ";
    cin >> base;
    cout << "Digite o valor do expoente: ";
    cin >> expoente;
    int resultado = potencia(base, expoente);
    cout << "Resultado: " << resultado << endl;

    return 0;
}

File: Unicatólica/Semestre_2025_1/POO - Programação Orientada a Objetos/poo_2025_1/Turma A/Códigos/revisao/src/revisao/Q1.java
package revisao;

import java.util.Scanner;

public class Q1 {
	public static void main(String[] args) {
		int base = 0, potencia = 0, resultado = 1;
		Scanner sc = new Scanner(System.in);
		System.out.println("Digite a base: ");
		base = sc.nextInt();
		System.out.println("Digite a potência: ");
		potencia = sc.nextInt();
		for(int i = 1; i <= potencia; i++) {
			resultado = resultado * base;
		}
		System.out.println("Resultado: " + resultado);
	}
}
